### PR TITLE
Add min OVN version required

### PR DESCRIPTION
**Pull request by vthapar**
Signed-off-by: Vishal Thapar <5137689+vthapar@users.noreply.github.com>

### CODE REVIEW

File: src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ create OVN resources necessary for Submariner, including:
 * A `submariner_join` logical switch that connects the `submariner_router`
   with the `ovn_cluster_router`.
 
+{{% notice info %}}
+Requires OVN NorthBound DB version 6.1.0+
```
Comment (sridhargaddam) [inline src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md]: Can we add that its available since OCP4.11+

Comment (vthapar) [reply src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md]: I used this coz it is more generic. OCP 4.11+ is too specific to OCP. I do want to mention it somewhere, but couldn't figure out good place to add it. But now that you mention it, additional comment about OCP version should be fine here.

**Review by sridhargaddam — APPROVED** [review]

**Comment by dfarrell07** [comment]
Could we add some note about why? What changed in this OVN version that we require? I know users have found that reasoning useful in the past, regarding our K8s version support.

**Comment by vthapar** [comment]
> Could we add some note about why? What changed in this OVN version that we require? I know users have found that reasoning useful in the past, regarding our K8s version support.

It is mainly to do with ttps://github.com/submariner-io/submariner/pull/1689 Older versions are broken due to the issue 1689 fixed and fix requires a version only available in one we mentioned. Any suggestions on how to word it here without giving details on the issue itself?

**Comment by dfarrell07** [comment]
> > Could we add some note about why? What changed in this OVN version that we require? I know users have found that reasoning useful in the past, regarding our K8s version support.
> 
> It is mainly to do with ttps://github.com/[submariner-io/submariner/pull/1689](https://github.com/submariner-io/submariner/pull/1689) Older versions are broken due to the issue 1689 fixed and fix requires a version only available in one we mentioned. Any suggestions on how to word it here without giving details on the issue itself?

Is the fundamental change the [move from `ebay/go-ovn` to `ovn-org/libovsdb`](https://github.com/submariner-io/submariner/pull/1689/commits/5a47c29782ef6142209e19b72fe80f96aed68d9c#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R13)?

**Review by tpantelis — APPROVED** [review]

**Comment by vthapar** [comment]
> > > Could we add some note about why? What changed in this OVN version that we require? I know users have found that reasoning useful in the past, regarding our K8s version support.
> > 
> > 
> > It is mainly to do with ttps://github.com/[submariner-io/submariner/pull/1689](https://github.com/submariner-io/submariner/pull/1689) Older versions are broken due to the issue 1689 fixed and fix requires a version only available in one we mentioned. Any suggestions on how to word it here without giving details on the issue itself?
> 
> Is the fundamental change the [move from `ebay/go-ovn` to `ovn-org/libovsdb`](https://github.com/submariner-io/submariner/pull/1689/commits/5a47c29782ef6142209e19b72fe80f96aed68d9c#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R13)?

Yes. Libovsdb doesn't work with older versions as it uses some fields which are only available in newer versions of schema.

**Pull request merged**